config/rbac/role.yaml

```diff
@@ -39,17 +39,7 @@ rules:
 - apiGroups:
   - ceph.rook.io
   resources:
-  - cephblockpools
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - update
-  - watch
-- apiGroups:
-  - ceph.rook.io
-  resources:
+  - CephBlockPoolRadosNamespace
```

Comment:
Nit: this could be `cephblockpoolradosnamespaces` to keep the capitalization and pluralization consistent with the rest of the resources.

Reply:
Updated